### DISCUSSION TITLE
Url from Deno.land/x

### DISCUSSION THREAD

**Discussion by garronej** [mxxii/leac#51] [Show and tell]
Hi @KillyMXI,  

Just to let you know that in the latest version of denoify we resolve, on a best effort basis, to deno.land/x urls.  

This mean that you'll get
```diff
-import { assert } from "https://raw.githubusercontent.com/mxxii/leac/v0.6.0/deno_dist/mod.ts";
+import { createLexer } from "https://deno.land/x/leac/v0.6.0/mod.ts";
```

**Comment by KillyMXI** [comment]
Awesome, thank you!

I can confirm it still works correctly with [parseley](https://github.com/mxxii/parseley) package after I remove explicit ports, even if I use the second entry point of `peberminta` there as well:

```diff
--- a/deno/parser.ts
+++ b/deno/parser.ts
@@ -1,6 +1,7 @@

-import { createLexer, Token } from 'https://deno.land/x/leac@v0.6.0/leac.ts';
-import * as p from 'https://deno.land/x/peberminta@v0.8.0/core.ts';
+import { createLexer, Token } from 'https://deno.land/x/leac@v0.6.0/mod.ts';
+import * as p from 'https://deno.land/x/peberminta@v0.8.0/mod.ts';
+import * as pc from 'https://deno.land/x/peberminta@v0.8.0/char.ts';
```

**Reply by garronej** [reply]
Great thanks for the feadback! (I submited the wrong diff but you got the gist)
